### PR TITLE
W-39 Add QA component

### PR DESCRIPTION
**Pull request by Mariii-23**
# Description

The main goal of this PR is to add a ``QA`` component.
Here we also add:

- ``Accordion Component`` :: for this we need to add a dependency of react : ["react-accessible-accordion" ](https://www.npmjs.com/package/react-accessible-accordion)

- ``QA json's file``
```typescript
interface IQAProps {
    question: string,
    answer: string
}

export interface IQAData {
    title: string,
    description: string,
    data: IQAProps[],
}
```
- Here we also add some gap between the ``tools``

image

image
image

image

## Issue
https://github.com/b1tsway/b1tsway.github.io/issues/39

## Type of change

Please delete options that are not relevant or select with a "x" the correct option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist(mark with a "x" the items you made):

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I tested all related functionalities and I didn´t break anything

### CODE REVIEW

File: src/components/atoms/Tools/styles.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
     height: 100%;
     display: grid;
     grid-template-columns: repeat(3, 1fr);
+    gap: 30px;
```
Comment (assebc) [inline src/components/atoms/Tools/styles.css:6]: 30/4 ;)

File: src/components/organisms/Accordion/styles.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+.accordion__item + .accordion__item {
+    border-top: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.accordion__button {
+    background-color: #eae3d7;
+    color: #008f8f;
+    cursor: pointer;
+    padding: 18px;
```
Comment (assebc) [inline src/components/organisms/Accordion/styles.css:19]: 18/4 ;)

**Review by assebc — CHANGES_REQUESTED** [review]
change css components for numbers that can be divided by 4 without any rest remaining!

File: src/components/atoms/Tools/styles.css (hunk 1 above)
Comment (Mariii-23) [reply src/components/atoms/Tools/styles.css]: Fix

File: src/components/organisms/Accordion/styles.css (hunk 2 above)
Comment (Mariii-23) [reply src/components/organisms/Accordion/styles.css]: fix

**Pull request merged**